src/10-serial-communication/nix-tooling.md

```diff
@@ -1,5 +1,25 @@
 # *nix tooling
 
+## Newer revisions of the discovery board
+
+With newer revisions, if you connect the discovery board to your laptop / PC you
+should see a new TTY device appear in `/dev`.
+
+``` console
```

Comment:
I'm okay with only having linux instructions here for now, but could you please open a ticket to add instructions for Windows/Mac as well?

Reply:
This is the "nix* tooling" section so this also applies to macOS. Windows is documented in "windows tooling".

Reply:
Except that no sane person would use `minicom` these days which also isn't shipped by default on most systems. 😏

Something like screen is a far more ubiquitous choice and is available on a lot of systems out of the box. Maybe we should add some instructions.